library/cpp/threading/local_executor/tbb_local_executor.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "local_executor.h"
+#define __TBB_TASK_ISOLATION 1
+
+#include <contrib/libs/tbb/include/tbb/blocked_range.h>
+#include <contrib/libs/tbb/include/tbb/parallel_for.h>
+#include <contrib/libs/tbb/include/tbb/task_scheduler_init.h>
+#include <contrib/libs/tbb/include/tbb/task_arena.h>
+#include <contrib/libs/tbb/include/tbb/task_group.h>
+
+namespace NPar {
+    class TTbbLocalExecutor: public TLocalExecutor {
```

Comment:
we have just added ILocalExecutor to local_executor.h .
please derive TTbbLocalExecutor from ILocalExecutor. 
this way there are only four methods to override.

Reply:
done